src/components/AdminSettings.vue

```diff
@@ -580,16 +585,10 @@ export default {
 		},
 		projectFolderSetUpErrorMessageDescription(errorKey) {
 			switch (errorKey) {
-			case 'The group folder name "OpenProject" integration already exists' :
-				return t('integration_openproject', 'Please make sure to rename the group folder or completely delete the previous one or deactivate the automatically managed folders.')
-			case 'The group folder app is not installed' :
-				return t('integration_openproject', 'Please install the group folder to be able to use automatic managed folders or deactivate the automatically managed folders.')
-			case 'The user "OpenProject" already exists' :
-				return t('integration_openproject', 'Please make sure to completely delete the previous user or deactivate the automatically managed folders.')
-			case 'The group "OpenProject" already exists' :
-				return t('integration_openproject', 'Please make sure to completely delete the previous group or deactivate the automatically managed folders.')
+			case 'The "Group folders" app is not installed' :
+				return t('integration_openproject', 'Please install the "Group folders" app to be able to use automatic managed folders or deactivate the automatically managed folders.')
```

Comment:
I think this should be `automatically`
```suggestion
				return t('integration_openproject', 'Please install the "Group folders" app to be able to use automatically managed folders or deactivate the automatically managed folders.')
```

Reply:
let's merge this PR we can make this change when we get information from other side